synapse/cmds/cortex.py

```diff
@@ -151,27 +151,3 @@ def nodevalu(t):
         self.printf('(%d results)' % (len(nodes),))
 
         return resp
-
-class NextSeqCmd(s_cli.Cmd):
```

Comment:
@therealsilence requested that I remove this and instead use the `ask` operator `nexttag`